code/renderers/svelte/template/cli/Button.svelte

```diff
@@ -9,7 +9,7 @@
   /**
    * What background color to use
    */
-  export let backgroundColor;
+  export let backgroundColor = undefined;
```

Comment:
This tells Svelte the prop is optional - removing an annoying console warning about the missing prop.